VContainer/Assets/VContainer/Runtime/Unity/PlayerLoopItem.cs

```diff
@@ -149,15 +150,15 @@ public bool MoveNext()
 
     sealed class FixedTickableLoopItem : IPlayerLoopItem, IDisposable
     {
-        readonly IEnumerable<IFixedTickable> entries;
+        readonly List<IFixedTickable> entries;
         readonly EntryPointExceptionHandler exceptionHandler;
         bool disposed;
 
         public FixedTickableLoopItem(
             IEnumerable<IFixedTickable> entries,
             EntryPointExceptionHandler exceptionHandler)
         {
-            this.entries = entries;
+            this.entries = entries.ToList();
```

Comment:
I think it would be better if `ToList()` was removed.
( Perhaps the copy occurs even if the concrete type is already a List.

This class is internal and has no use case other than being passed an `IReadOnlyList<T>`.
In this case, why not keep `IReadOnlyList<T>` as an argument, and use `for` and `[]` operator instead of `foreach` ?
( `for ...` statement  is exactly the same as the implementation of List<>.GetEnumerator. )

Reply:
>In this case, why not keep IReadOnlyList<T> as an argument, and use for and [] operator instead of foreach ?

I like that!